Makefile

```diff
@@ -15,20 +20,20 @@ update-tools:
 	@ echo " >> Tools Updated << "
 
 # Development targets
-install:
+install: .tools/cluster.Makefile
```

Comment:
nit: Should this instead depend on dev.Makefile ? 

It should still work 99.9999% of the time, as long as nothing creative enough happens.